openfisca_aotearoa/variables/entitlements/income_tax/working_for_families.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_aotearoa.entities import Person, Family
+
+
+class social_security__received_income_tested_benefit(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    label = u'Boolean for if a Person is classified as receiving an income tested benefit'
+    reference = "http://www.legislation.govt.nz/act/public/2007/0097/latest/whole.html#DLM1518484"
+
+
+class veterans_support__received_parents_allowance(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    label = u'Boolean for if a Person is classified as receiving a parents allowance'
+    reference = "http://www.legislation.govt.nz/act/public/2007/0097/latest/whole.html#DLM1518484"
+
+
+class veterans_support__received_childrens_pension(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    label = u'Boolean for if a Person is classified as receiving a parents allowance'
+    reference = "http://www.legislation.govt.nz/act/public/2007/0097/latest/whole.html#DLM1518484"
+
+
+class income_tax__residence(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    default_value = True  # TODO residency is required as defined in the act
+    label = u'Boolean for if a Person is classified as meeting residence requirements'
+    reference = "http://www.legislation.govt.nz/act/public/2007/0097/latest/DLM1518482.html"
+    # This should really be a forumla based variable covering the full residency criteria.
```

Comment:
> This comment is not useful as it expresses frustration or wishful thinking rather than helping the reader understand the code.
> 
> Possible options are to remove it altogether, or to amend it into something like “Mocked as an input variable until all residency criteria are implemented.”
> I am not certain  of the value of the latter statement, as this can be true of almost all modelled elements.